api/service/prometheus/service.go

```diff
@@ -99,3 +172,43 @@ func (s *Service) Status() error {
 	}
 	return nil
 }
+
+// SetConfig initialize the prometheus config
+func SetConfig(
```

Comment:
Too many arguments. For common practice, we shall not let number of args larger than 4. It would be better to make Config exported and initialized in New.

Reply:
okay, will export all config variable and pass in the variable then.